normalize_all.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# For loop to normalize files from each data source for each language.
+
+for language in af mg
```

Comment:
Does this also not include all the languages?

Reply:
Definitely an oversight! This was my minimal working example and then I forgot to go back in and add the rest.

Reply:
Nice!